src/Backend/Routes/AuthProviders/FacebookAuth.js

```diff
@@ -0,0 +1,21 @@
+import { getAuth, signInWithPopup, FacebookAuthProvider } from "firebase/auth";
+
+const auth = getAuth();
+signInWithPopup(auth, provider)
```

Comment:
In general lots of unused consts. I wouldn't even have the error caught in this function, have error handling at a UI level so you can popup a toast or something

Reply:
Also does this just run when file is iported? You should be exporting a function called `loginWithFacebook` or something